test/python/transpiler/test_basis_translator.py

```diff
@@ -816,3 +816,42 @@ def test_condition_set_substitute_node(self):
         expected.measure(1, 1)
         expected.u2(0, pi, 0).c_if(cr, 1)
         self.assertEqual(circ_transpiled, expected)
+
+    def test_skip_target_basis_equivalences_1(self):
+        """Test that BasisTranslator skips gates in the target_basis - #6085"""
+        from qiskit.test.mock import FakeAthens
```

Comment:
Is there a reason to do this as a runtime import vs just putting it at the top and importing it with the module?

Reply:
Not really. ;-) Done in https://github.com/Qiskit/qiskit-terra/pull/6369/commits/0cb1626037a4ba784cb435e03e36c80ce6b3d4b2.